src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -411,6 +411,7 @@ def load_arguments(self, _):
         # managed cluster
         c.argument('http_proxy_config')
         c.argument('load_balancer_managed_outbound_ipv6_count', type=int)
+        c.argument('outbound_type', arg_type=get_enum_type(outbound_types))
```

Comment:
```suggestion
        c.argument('outbound_type', arg_type=get_enum_type(outbound_types, "loadbalancer"))
```

Reply:
this option can be nil and it will not update outbound type when it is nil